### ISSUE TITLE
-ms-transform is not being removed web config exclude IE support

### ISSUE THREAD

**Issue by pkuczynski** [postcss/autoprefixer#404]
My ```browserslist``` looks as following:

```
last 2 Chrome versions
last 2 Opera versions
last 2 Firefox versions
```

I am intentionally excluding IE, as we do not want to support it. Still my file contains lines like

```css
@-ms-keyframes spin {
    0% {
        -ms-transform: rotate(0deg)
    }

    100% {
        -ms-transform: rotate(359deg)
    }
}
```

which come from Font Awesome 3.2.1. How do I get rid of those?

**Comment by ai** [comment]
Awesome! :facepalm:

`-ms-keyframes` and `-ms-transform` never existed. I just love this ghost prefixes :D.

**Comment by ai** [comment]
I will add them to misteakes list.

**Comment by ai** [comment]
Done https://github.com/postcss/autoprefixer-core/commit/b55c24a2f771b62bbe4532dd1d62f68a179480e9

**Comment by ai** [comment]
Released in 5.1.2

**Comment by pkuczynski** [comment]
I actually found four more and reported in #405. Sorry for doing it as a separate issue, haven't notice them in the first place...
